pyiron_contrib/protocol/generic.py

```diff
@@ -333,14 +333,13 @@ def command(self, *args, **kwargs):
         """The command method controls the physics"""
         pass
 
-    def execute_parallel(self, queue, n, input_dict):
+    def execute_parallel(self, n, return_dict):
         """How to execute in parallel when there's a list of these vertices together."""
-        output_data = self.command(**input_dict)
-        queue.put((n, output_data))
-        # Note: The output needs to be explicitly collected and archived later if this is used in place of `execute`
+        output_data = self.command(**self.input.resolve())
+        return_dict[n] = output_data
```

Comment:
I believe it is a dict. Since I define it like this in `list.py`:

```
from multiprocessing import Manager

manager = Manager()
return_dict = manager.dict()
```
And then populate it, with the output data of each image in the order of their collection after parallel execution.